system/Files/File.php

```diff
@@ -70,17 +70,43 @@ public function getSize()
         return $this->size ?? ($this->size = parent::getSize());
     }
 
+    /**
+     * Retrieve the file size by unit, calculated in IEC standards with 1024 as base value.
+     *
+     * @phpstan-param positive-int $precision
+     *
+     * @return false|int|string
+     */
+    public function getSizeByUnitBinary(FileSizeUnit $unit = FileSizeUnit::B, int $precision = 3)
```

Comment:
this is a nitpick, so open for discussion. would this method and its metric counterpart be renamed as `getSizeByBinaryUnit` and `getSizeByMetricUnit` so that the English flows more naturally?

Reply:
I like this better as well, gonna change it =)